web/rest/user/config/api/raw/provider.yml

```diff
@@ -386,3 +386,44 @@ Ivoz\Provider\Domain\Model\Fax\Fax:
     swagger_context:
       required:
         - company
+
+Ivoz\Provider\Domain\Model\Recording\Recording:
+  itemOperations:
+    get:
+      method: 'GET'
+      path: '/recordings/{id}'
+      requirements:
+        id: '\d+'
+    delete: ~
+  collectionOperations:
+    get: ~
+    get_multiple_recorderded_files:
+      method: 'GET'
+      path: '/recordings/recorded_files_zip'
+      route_name: 'get_multiple_recorderded_files'
```

Comment:
Normally we try to resemble the names of route_name and path

    get_recorderded_files_zip:
      method: 'GET'
      path: '/recordings/recorded_files_zip'
      route_name: 'get_recorderded_files_zip'

Reply:
That name was not my choice: it's already called so in client portal and IMHO it should be named the same in both sites.

https://github.com/irontec/ivozprovider/blob/fb642fab38f65342d3b9bf6fb9dd863f5961fcb6/web/rest/client/config/api/raw/provider.yml#L1754